bob-apps/bin/blob_recovery_tool/key.rs

```diff
@@ -0,0 +1,64 @@
+use super::prelude::*;
+
+macro_rules! sized_key {
+    ($t:ident, $n:expr) => {
+        #[derive(Clone, Debug, PartialEq, Eq)]
+        pub struct $t(Vec<u8>);
```

Comment:
Key trait was updated in `https://github.com/qoollo/bob/pull/397` (`RefKey` was added), so this macro should be updated accordingly